plugins/filter/jc.py

```diff
@@ -42,6 +42,12 @@
 '''
```

Comment:
The previous line (requirements) should be expanded to clarify to users that they have to `pip install jc`.

Reply:
The `local Ansible controller` part is already shown in the docs, it shouldn't be added here again. See https://docs.ansible.com/ansible/devel/collections/community/general/jc_filter.html#requirements.